### PR TITLE
Fog Ramp Path

### LINKED ISSUE #530: i need fog sphere
### Feature

it'd be cool i swear, even though im the only one that noticed this in 1 year

timber hearth has a special fog sphere that makes it blue on the front

### Context

fact
i need

### PR DESCRIPTION
**Pull request by JohnCorby**
## Major features
-

## Minor features
- Add support for custom `fogColorRampTexture` (resolves #530)

## Improvements
-

## Bug fixes
-

### CODE REVIEW

**Comment by JohnCorby** [comment]
classic tested this and it "didnt work" but i put that in quotes because i dont trust classic

**Comment by JohnCorby** [comment]
closing for now. classic appears to not need this feature, nor does anyone else

**Comment by xen-42** [comment]
I say let's just merge it

**Review by xen-42 — APPROVED** [review]

File: NewHorizons/Builder/Atmosphere/FogBuilder.cs [hunk 1]
```diff
@@ -59,6 +60,8 @@ public static PlanetaryFogController Make(GameObject planetGO, Sector sector, At
             PFC.fogDensity = atmo.fogDensity;
             PFC.fogExponent = 1f;
             var colorRampTexture = atmo.fogTint == null ? _ramp : ImageUtilities.TintImage(_ramp, atmo.fogTint.ToColor());
```
Comment (xen-42) [inline NewHorizons/Builder/Atmosphere/FogBuilder.cs:62]: Wait yeah don't tint just do an if else

**Review by xen-42 — CHANGES_REQUESTED** [review]

**Review by xen-42 — APPROVED** [review]

**Pull request merged**